SparseGrids/tasgridUnitTests.cpp

```diff
@@ -34,6 +34,8 @@
 #include "tasgridUnitTests.hpp"
 #include "tasgridExternalTests.hpp"
 #include "tasgridTestHelpers.hpp"
+#include "tsgBlasWrappers.hpp"
```

Comment:
Somewhere in the BLAS wrappers, there should be pre-processor clause that guards in the case when BLAS is not enabled. Make sure it works with and without BLAS.

Reply:
Wouldn't it be better to use pre-processor directives in tasgridUnitTest.cpp (see the latest commit) itself?

Reply:
I also tested the latest commit with Tasmanian_ENABLE_BLAS=OFF and it seems to be working fine.

Reply:
The main point is that this should work with `Tasmanian_ENABLE_BLAS` and don't want the C++ wrappers to be visible when they refer to LAPACK functions that cannot be linked. Look at `tsgTPLWrappers.hpp` and perhaps you should include that file.

Reply:
See the comment below about `#ifdef Tasmanian_ENABLE_BLAS`.

Reply:
Done. I replaced the conditional import of ``tsgBlasWrappers.hpp`` with an import of ``tsgTPLWrapper.hpp``.